src/conditionals/admin/post-quick-edit-conditional.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Yoast\WP\SEO\Conditionals\Admin;
+
+use Yoast\WP\SEO\Conditionals\Conditional;
+
+/**
+ * Class Post_Quick_Edit_Conditional
+ */
+class Post_Quick_Edit_Conditional implements Conditional {
```

Comment:
kun je de naamgeving nog iets aanscherpen? ik mis een werkwoord. Is Editing? Saving?